app/models/user.rb

```diff
@@ -74,6 +75,10 @@ def self.authenticate(login, password)
       pipe.puts("#{login}\n#{password}") # write to stdin of markus_config_validate
       pipe.close
       m_logger = MarkusLogger.instance
+      if (defined? VALIDATE_CUSTOM_EXIT_STATUS) && $?.exitstatus == VALIDATE_CUSTOM_EXIT_STATUS
+        m_logger.log("Login failed. Reason: Custom exit status.", MarkusLogger::ERROR)
```

Comment:
Line is too long. [86/80]<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.